src/GraphQL/Execution/Nodes/ExecutionNode.cs

```diff
@@ -55,7 +55,19 @@ public abstract class ExecutionNode
         /// <summary>
         /// Returns the AST field alias, if specified, or AST field name otherwise.
         /// </summary>
-        public string? Name => Field?.Alias ?? Field?.Name;
+        public string? Name => (string)(Field?.Alias?.Name ?? Field?.Name); //TODO:alloc
+
+        private object GetNameOrAlias(bool preferAlias)
```

Comment:
Looks like this is returning `object` because it's in the `pathlist` array which is a `object[]`.  In that case, using a ROM has almost no benefit.  A string can store 6 characters in 24 bytes, and 10 characters in 32 bytes.  A boxed ROM is always 30 bytes.  (Needs verification).  In either case, an allocation occurs.  Just pointing out that the ROMJsonConverter probably is pretty pointless, especially as this is only effective on the error path.  I think the API would be better to eliminate the ROMJsonConverter and just return strings and ints in the path property like we have been doing.

Reply:
I did not finish with this and I will think about something.

Reply:
ROMJsonConverter removed